api-classes.js

```diff
@@ -47,6 +47,28 @@ class StoryList {
     // TODO - Implement this functions!
     // this function should return the newly created story so it can be used in
     // the script.js file where it will be appended to the DOM
+    const token = user.token
+    const author = newStory.author
+    const title = newStory.title
+    const url = newStory.url
+
+    const response = await axios.post(`${BASE_URL}/stories`, {
+      token,
+      story: {
+        author,
+        title,
+        url
+      }
+    })
+    return response.data.story
+  }
+
+  async removeStory(storyId) {
+    const token = localStorage.token
+    const response = await axios.delete(`${BASE_URL}/stories/${storyId}`, {
+      token
+    })
+    return response.message
```

Comment:
Hi @Lexprono , there should be not problem with the `delete` method of axios. 
Could you get a screenshot of the error you're getting when you try to delete a story?
The URL seems fine so maybe check the value of the token and verify that's the correct one

Reply:
![Image 2-3-21 at 12 01 PM](https://user-images.githubusercontent.com/76506172/106789673-333fce80-6618-11eb-83ca-1596bc3e833d.jpg)

I believe it should be attached. I also just ran it again with the token logged and it is correct.